meta-rzg2l/docs/template/conf/rzpi/local.conf.sample

```diff
@@ -274,9 +274,9 @@ SDKIMAGE_FEATURES_append = " staticdev-pkgs dev-pkgs dbg-pkgs"
 # Enable pam distro feature
 DISTRO_FEATURES_append = " pam"
 
-# Enable wayland and disable x11
+# Enable wayland and enable x11
 DISTRO_FEATURES_append = " wayland"
```

Comment:
> Are you keeping both X11 and wayland?

Yes, since Tkinter requires Python 3 to be built with Tk. The Tk bitbake file from meta-openembedded specifies that it depends on X11: https://git.openembedded.org/meta-openembedded/tree/meta-oe/recipes-devtools/tcltk/tk_8.6.10.bb?h=dunfell

``` 
inherit autotools features_check
# depends on virtual/libx11
REQUIRED_DISTRO_FEATURES = "x11"

EXTRA_OECONF = "\
    --enable-threads \
    --with-x \
    --with-tcl=${STAGING_BINDIR}/crossscripts \
    --libdir=${libdir} \
"
export TK_LIBRARY='${libdir}/tk${VER}'
```

So  we need to keep both x11 and wayland

Reply:
Below is the log I took a part from this file: /var/log/weston.log
- Only wayland feature:
```
Command line: /usr/bin/weston --idle-time=0 --log=/var/log/weston.log
```
- Enable both wayland and X11
```
Command line: /usr/bin/weston --idle-time=0 --debug --modules=xwayland.so --log=/var/log/weston.log
```
So it enables Xwayland, which allows X11 applications to run in a Wayland environment => we do support both wayland and X11.